src/log.h

```diff
@@ -0,0 +1,28 @@
+#ifndef BENCHMARK_LOG_H_
+#define BENCHMARK_LOG_H_
+
+#include <ostream>
+
+namespace benchmark {
+namespace internal {
+
+int GetLogLevel();
```

Comment:
I'll make the change. Any input for what to call the levels?

FYI the reason I didn't use an enum is because the log level is set via the `-v=<int>` flag. So I figured there was no point obfuscating that input behind an enum.